src/types/Tag.ts

```diff
@@ -14,3 +15,12 @@ export default interface Tag extends CreatedUpdatedProps {
   default?: boolean
   deleted?: boolean
 }
+
+export interface ImportedTag {
+  id?: string;
+  description?: string;
+  importedBy: string;
+  importedOn?: Date;
+  error?: string;
```

Comment:
Error should be an enum

Reply:
I removed this property and replaced it with errorCode / errorDescription